app/scripts/background.js

```diff
@@ -885,12 +884,16 @@ export function setupController(
         senderUrl?.origin === `chrome-extension://${browser.runtime.id}`;
     }
 
+    controller.remoteFeatureFlagController.getRemoteFeatureFlags();
```

Comment:
we will need error handling here now

Reply:
and should we remove the `controller.remoteFeatureFlagController.getRemoteFeatureFlags()` call on 887, because we have one on 896?

Reply:
Yes this line is removed, it was for testing purpose 🙏🏻

Reply:
Error handling within `initializeRemoteFeatureFlags` can be found 5204e9c4ff436e00a8f512319bcf0c23b88d6393